code/app/src/main/res/layout/activity_main.xml

```diff
@@ -19,4 +19,12 @@
         android:layout_marginTop="600dp"
         android:layout_marginLeft="150dp"/>
 
+    <Button
+        android:id="@+id/storage_activity2"
```

Comment:
Use a more intuitive id to depict what the button is for!

Reply:
this is just for testing, not for use